blog/welcome-to-developer-development/index.qmd

```diff
@@ -11,3 +11,43 @@ page-layout: full
 The Developer Development
 
 {{< include /_back-blog.qmd >}}
+
+# Week 1
+
+## SE1
+
+#### Summary
+
+In the preface of *Software Engineering at Google*, the section [Programming Over Time](https://abseil.io/resources/swe-book/html/pr01.html#programming_over_time) defines software engineering as "programming integrated over time." The purpose of the book is to answer reflective questions that Software Engineering (SE) teams should have during the life cycle of a project and a company or institution. The thesis of the book is then laid out as motivated by three principles that should be held in the development process: **Time and Change**, **Scale and Growth**, and **Trade-offs and Costs**. We will work hard to adapt to the changes made by our peers and to be the most amiable as a software development team.
+
+#### Reflection
+
+<!--
+Needs to be added too
```

Comment:
This is the reflection that I have written for SE1; it could possibly fill this slot for the reflection section.

The preface and programming over time sections outline the foundation for the book’s exploration of software engineering with insights from Google’s unique perspective. I believe this will give us an understanding of the material and a look into how the industry, particularly Google, looks at software engineering and what level of rigor they expect.